src/__tests__/tic-tac-toe.test.ts

```diff
@@ -1,49 +1,49 @@
-import Game from "../tic-tac-toe";
+import Game from '../tic-tac-toe';
 
-describe("Tic-tac-toe", () => {
+describe('Tic-tac-toe', () => {
   let game: Game;
 
   beforeEach(() => {
     game = new Game();
   });
 
-  it("will generate an empty board that is 3x3 by default", () => {
+  it('will generate an empty board that is 3x3 by default', () => {
     const board = [
       [undefined, undefined, undefined],
       [undefined, undefined, undefined],
-      [undefined, undefined, undefined]
+      [undefined, undefined, undefined],
     ];
     expect(game.board).toEqual(board);
   });
 
-  it("will generate a 5x5 board given the correct arguments", () => {
+  it('will generate a 5x5 board given the correct arguments', () => {
     const board = [
       [undefined, undefined, undefined, undefined, undefined],
       [undefined, undefined, undefined, undefined, undefined],
       [undefined, undefined, undefined, undefined, undefined],
       [undefined, undefined, undefined, undefined, undefined],
-      [undefined, undefined, undefined, undefined, undefined]
+      [undefined, undefined, undefined, undefined, undefined],
```

Comment:
In the future, check out `Array.fill` as a way to dry these up: `Array(5).fill(undefined).map(() => Array(5).fill(undefined))`